examples/notebooks/correction_walkthrough.ipynb

```diff
@@ -202,19 +269,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "new_curv_correction = da.CurvatureCorrection(config_source = \"../images/config.json\")\n",
+    "new_curv_correction = da.CurvatureCorrection(config = \"../images/config.json\")\n",
```

Comment:
I find it always more appealining to write out paths only once, since they look pretty ugly and are prone to errors. Instead storing and resuing the variable names, in my opinoin, looks much cleaner and easier to read.